Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Home/HomeTabBarController.swift

```diff
@@ -8,22 +8,39 @@
 import UIKit
 
 class HomeTabBarController: UITabBarController {
-
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
+        
+        let storyboard = UIStoryboard(name: "GradesView", bundle: nil)
+        let gradesViewController = storyboard.instantiateViewController(withIdentifier: "GradesViewController")
+        
+        let navController = UINavigationController()
+        let coursesViewController = CoursesViewController.loadFromNib()
+        navController.pushViewController(coursesViewController, animated: false)
+        let profileViewController = ProfileViewController()
+        
+        self.setViewControllers([ gradesViewController,navController, profileViewController], animated: false)
+        setTabBarItems()
     }
     
-
-    /*
-    // MARK: - Navigation
-
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        // Get the new view controller using segue.destination.
-        // Pass the selected object to the new view controller.
+    func setTabBarItems(){
+        let gradesViewController = (self.tabBar.items?[0])! as UITabBarItem
+        gradesViewController.image = UIImage(named: "grades")?.withRenderingMode(UIImage.RenderingMode.alwaysOriginal)
+        gradesViewController.selectedImage = UIImage(named: "Selected ")?.withRenderingMode(UIImage.RenderingMode.alwaysOriginal)
+        gradesViewController.title = ""
+        gradesViewController.imageInsets = UIEdgeInsets(top: 6, left: 0, bottom: -6, right: 0)
+        
+        let coursesViewController = (self.tabBar.items?[1])! as UITabBarItem
+        coursesViewController.image = UIImage(named: "courses")?.withRenderingMode(UIImage.RenderingMode.alwaysOriginal)
+        coursesViewController.selectedImage = UIImage(named: "Selected")?.withRenderingMode(UIImage.RenderingMode.alwaysOriginal)
+        coursesViewController.title = ""
+        coursesViewController.imageInsets = UIEdgeInsets(top: 6, left: 0, bottom: -6, right: 0)
+        
+        let profileViewController = (self.tabBar.items?[2])! as UITabBarItem
+        profileViewController.image = UIImage(named: "profile")?.withRenderingMode(UIImage.RenderingMode.alwaysOriginal)
+        profileViewController.selectedImage = UIImage(named: "Selected")?.withRenderingMode(UIImage.RenderingMode.alwaysOriginal)
+        profileViewController.title = ""
+        profileViewController.imageInsets = UIEdgeInsets(top: 6, left: 0, bottom: -6, right: 0)
```

Comment:
Some improvements can be done here. Create a `setupTabBarItem` function does all the required configuration and use it for all the 3 tab bar items to avoid duplicate code.
Also, why do the name of the variables have "ViewController" as a suffix?